webapp/components/customTunnelsThroughPartner/index.tsx

```diff
@@ -47,14 +47,18 @@ export const CustomTunnelsThroughPartner = function ({
             {t(`${operation}.subheading`, { symbol: fromToken.symbol })}
           </DrawerParagraph>
         </div>
-        <Stargate fromToken={fromToken} toToken={toToken} />
+        <div>
```

Comment:
And these...

Reply:
See https://github.com/hemilabs/ui-monorepo/pull/789#discussion_r1930939579